tools/docker/env/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y -q --no-install-recommends
 	# These are needed to build Linux kernel:
 	flex bison bc libelf-dev libssl-dev \
 	# qemu-user is required to run alien arch binaries in pkg/cover tests.
-	qemu-user qemu-user-binfmt \
```

Comment:
Something is wrong here.

Reply:
It is from https://github.com/google/syzkaller/pull/5575/commits, can you approve that one before?